srcpkgs/babashka/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'babashka'
+pkgname=babashka
+version=0.0.88
+revision=2
```

Comment:
This refers to the template's revision not upstream's, so this should start at 1.

Reply:
The version is not allowed to have underscores.
How would I model something like this in the template?